src/points/PointsHome.tsx

```diff
@@ -145,7 +145,6 @@ export default function PointsHome({ route, navigation }: Props) {
                 onPress={onPressActivity}
                 text={t('points.activity')}
                 type={BtnTypes.GRAY_WITH_BORDER}
-                fontStyle={typeScale.labelXSmall}
```

Comment:
I think we may want to leave the font style as an option because of this. cc @kathaypacific

Reply:
naw if it's just for points, i think we can remove it. i'm a fan of not having customisable styles, they can be convenient but they add extra complexity and facilitate inconsistency